src/ui-manager.js

```diff
@@ -39,7 +39,7 @@ class UIManager {
   store: any;
   container: ?HTMLElement;
   root: React$Component<any, any, any>;
-  _translations: {[langKey: string]: Object} = {en: en_translations};
+  _translations: {[langKey: string]: Object} = {en: en_translations['en']};
```

Comment:
there is no need for the extra nesting IMO, the file is already called by the key name

Reply:
else the bundle builder should add the key name to the generated i18n json.